### PR TITLE
Shot in the dark

### PR DESCRIPTION
**Pull request by harlantwood**
This fixed failures on my machine.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -185,7 +185,7 @@ module.exports = {
     }
     freeport(function (err, port) {
       if (err) return cb(err)
-      opts['Addresses.Gateway'] = '""'
```
Comment (jbenet) [inline index.js]: that's very odd. wonder if it was here for a reason. @krl @travisperson ?

Comment (travisperson) [inline index.js]: Not that I'm aware of.

**Comment by krl** [comment]
Hmm, this was to not have the disposable nodes setup gateways

**Comment by harlantwood** [comment]
I don't claim to understand the nuances here.  But I do know the tests were failing before this commit, and passing after it: https://circleci.com/gh/harlantwood/node-ipfsd-ctl

**Comment by whyrusleeping** [comment]
@krl why was it being set to double empty quotes?

**Comment by krl** [comment]
Jeromy Johnson <notifications@github.com> writes:

> @krl why was it being set to double empty quotes?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ipfs/node-ipfsd-ctl/pull/5#issuecomment-112560667

It was about not being able to pass an empty string to the config, in
the way it was written before.

**Pull request merged**